.github/workflows/toc.yml

```
@@ -14,4 +14,4 @@ jobs:
         # Git commit name.
         COMMIT_NAME: "TOCGEN"
         CREATE_PR: false
-        TARGET_PATHS: "./[A-Z]*[A-Z]-[A-Z]*[A-Z]/README.md"
+        TARGET_PATHS: "./[A-Z]*[A-Z]-[A-Z]*[A-Z]/*.md"
```

```suggestion
        TARGET_PATHS: "./[A-Z]*[A-Z]-[A-Z]*[A-Z]/**.md"
```
@nukerxy @ZeroPointMax Oder so? Gibt es einen konkreten Fall wie wir das brauchen?

---

ähm das matcht dann rekursiv oder?

---

> ähm das matcht dann rekursiv oder?

Ja genau, anders werden die files im `extra`-Ordner mMn nicht gematched.

---

```suggestion
        TARGET_PATHS: "./[A-Z]*[A-Z]-[A-Z]*[A-Z]/README.md,./[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/*.md"
```

So würde mir das aber am besten gefallen^^

---

> > ähm das matcht dann rekursiv oder?
> 
> Ja genau, anders werden die files im `extra`-Ordner mMn nicht gematched.
> 

stehe auf dem Schlauch - wieso ausgerechnet extra, was ist mit appendix, assets; was ist der Unterschied?

---

```suggestion
        TARGET_PATHS: "./[A-Z]*[A-Z]-[A-Z]*[A-Z]/README.md,./[A-Z]*[A-Z]-[A-Z]*[A-Z]/appendix/*.md"
```
@nukerxy Tut mir Leid, ich habe das hier selbst etwas durcheinandergehauen. 

Der TOC macht mMn nur für den `appendix` Sinn. Hier nochmal die Erläuterung der Ordnerstruktur:

```text
<Modul>/
├── appendix/           <- Anhang: Praktika, Glossar, ...
│   ├── <Praktika>.md
│   ├── <Cheatsheet>.md
│   └── <Glossar>.md
├── extra/              <- Zusätzliches: Vorträge, Programmierübungen ...
│   ├── <Vortag>.pdf
│   └── excercise-01/
│       └── main.c
├── assets/             <- Bilder
│   ├── <filename>.png
│   ├── <filename>.png
│   └── <filename>.svg
└── README.md           <- Modulmitschriften
```